molgenis-data-rest/src/main/java/org/molgenis/data/rest/v2/RestControllerV2.java

```diff
@@ -283,7 +292,14 @@ public EntityCollectionBatchCreateResponseBodyV2 createEntities(@PathVariable("e
 			final List<String> ids = new ArrayList<String>();
 
 			// Add all entities
-			this.dataService.add(entityName, entities.stream());
+			if (ATTRIBUTE_META_DATA.equals(entityName))
```

Comment:
Metadata == data, the REST controller shouldn't make any entity specific exceptions.